kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -42,42 +42,43 @@ define([
     'use strict';
     return new KBWidget({
         name: 'StagingAreaViewer',
-
+  
```

Comment:
sorry for all the whitespace changes. the "real" party gets started down on line 374